pygluu/kubernetes/templates/helm/gluu/charts/config/templates/configmaps.yaml

```diff
@@ -389,8 +389,8 @@ data:
     def backup(hosts):
         timenow = time.strftime("%c")
         timestamp = "Backup occurred %s \n" % timenow
-        logger.info("Backing up hosts file to /etc/hosts.back ...")
-        with open('/etc/hosts.back', 'a+') as f:
+        logger.info("Backing up hosts file to /etc/gluu/hosts.back ...")
+        with open('/etc/gluu/hosts.back', 'a+') as f:
```

Comment:
Is this related to https://github.com/GluuFederation/cloud-native-edition/issues/577?

Reply:
Yes

Reply:
We can revert the changes on this file. I think the issue lies in our OCI images (they didn't specify path and permissions to `hosts.back` file)